examples/anonymous-function/anonymous_function.metatags

```diff
@@ -0,0 +1,2 @@
+description: This BBE demonstrates creating an arrow function, Lambda function, and anonymous function in Ballerina.
```

Comment:
When I am reading "which also commonly known an arrow function, Lambda function" part, i dont feel like we are describing about anonymous functions, because there is "in Ballerina" part after that.

Reply:
what about "This BBE demonstrates creating an anonymous function, which is commonly known as an arrow function or lambda function in Ballerina"?

Reply:
Resolving the discussion with this change https://github.com/ballerina-platform/ballerina-distribution/pull/3840#discussion_r1055409272